.github/workflows/build_brltty_windows.yml

```diff
@@ -0,0 +1,88 @@
+name: build BRLTTY for Windows
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    branches: [master]
+
+  workflow_dispatch:
+
+jobs:
+  build_brltty:
+    runs-on: windows-2025
```

Comment:
You could also use windows-latest here. The risk is that it breaks when the latest reference is updated to a newer version.